client/packages/programs/src/api/api.ts

```diff
@@ -258,7 +259,7 @@ export const getAllocateProgramNumber = (sdk: Sdk, storeId: string) => ({
 
 export type ProgramEnrolmentListParams = {
   sortBy?: SortRule<ProgramEnrolmentSortFieldInput>;
-  filterBy?: FilterBy;
+  filterBy?: ProgramEnrolmentFilterInput;
```

Comment:
Using this type from the generated API gives the correct autocompletion, type checks, etc...